internal/cmd/secrets-manager/instance/create/create.go

```diff
@@ -65,14 +71,25 @@ func NewCmd() *cobra.Command {
 				}
 			}
 
-			// Call API
-			req := buildRequest(ctx, model, apiClient)
+			// Call API to create instance
+			req := buildCreateInstanceRequest(ctx, model, apiClient)
 			resp, err := req.Execute()
 			if err != nil {
 				return fmt.Errorf("create Secrets Manager instance: %w", err)
 			}
 			instanceId := *resp.Id
 
+			// Call API to create ACls for instance, if ACL is provided
+			if model.Acls != nil && len(*model.Acls) > 0 {
+				requests := buildCreateACLRequests(ctx, model, instanceId, apiClient)
+				for _, req := range requests {
+					_, err := req.Execute()
+					if err != nil {
+						return fmt.Errorf("create ACL for Secrets Manager instance: %w", err)
+					}
+				}
```

Comment:
And would simplify a bit the error handling here

Reply:
As it is, it might not be entirely clear to the user in which state he ends up if e.g. the second call to create an ACL fails

Reply:
Nevertheless, we should still signal the user that the instance was created even if the call to create/update the ACLs failed.

The output could something like:
```shell
Error: create ACLs for Secrets Manager instance.

The creation of the ACLs for the Secrets Manager instance failed, but the instance was correctly created.
If you want to retry configuring the ACLs, you can do it via:
  $ stackit secrets-manager instance update (...)
```

Reply:
Alternatively, we could delete the newly created instance. What do you think @vicentepinto98?